flow/project.py

```diff
@@ -1294,7 +1288,8 @@ def _get_run_ops(ignore_ops, additional_ignores_flag):
             self.name,
             jobs,
             cmd=unevaluated_cmd,
-            directives=submission_directives,
+            directives={**submission_directives},
```

Comment:
I would prefer a constructor call rather than a comprehension here, since there's no "comprehending" to be done.
```suggestion
            directives=dict(submission_directives),
```